<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkPythonSnippets</class>
 <widget class="QWidget" name="QmitkPythonSnippets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>436</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="Name">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Content:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="PasteNow">
     <property name="text">
      <string>Paste Now!</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QPushButton" name="AddNewSnippet">
     <property name="text">
      <string>Add new Snippet</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="RemoveSnippet">
     <property name="text">
      <string>Remove Snippet</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QPushButton" name="RenameSnippet">
     <property name="text">
      <string>Rename Snippet</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QPushButton" name="RestoreDefaultSnippets">
     <property name="text">
      <string>Restore default Snippets</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QTextEdit" name="Content"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
